<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QPushButton" name="enter_button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Fax</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-style:outset;
	border-width:2px;
	border-radius:10px;
	border-color:black;
}

QPushButton:hover{
	border-style:outset;
	border-width:2px;
	border-radius:10px;
	border-color:black;
	border-color:rgb(51, 143, 255);
	color: rgb(51, 143, 255);
}</string>
     </property>
     <property name="text">
      <string>CONFIRM</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title_input">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Fax</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: white;
	border-style: outset;
	border-width: 1px;
	border-radius: 5px;
	padding-left: 20px;
	padding-right: 20px;
}

QLineEdit:hover{
	border-width: 2px;
	border-color:rgb(51, 143, 255);
}

QLineEdit:focus{
	border-width: 2px;
	border-color:rgb(51, 143, 255);
}</string>
     </property>
     <property name="placeholderText">
      <string>Book title</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Fax</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NEW TITLE</string>
     </property>
    </widget>
    <widget class="QLabel" name="background_label">
     <property name="geometry">
      <rect>
       <x>-180</x>
       <y>-110</y>
       <width>501</width>
       <height>361</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>501</width>
       <height>361</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>501</width>
       <height>361</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Source_imgs/bckgrnd_01.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_input">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Fax</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: white;
	border-style: outset;
	border-width: 1px;
	border-radius: 5px;
	padding-left: 20px;
	padding-right: 20px;
}

QLineEdit:hover{
	border-width: 2px;
	border-color:rgb(51, 143, 255);
}

QLineEdit:focus{
	border-width: 2px;
	border-color:rgb(51, 143, 255);
}</string>
     </property>
     <property name="placeholderText">
      <string>Book ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Fax</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <zorder>background_label</zorder>
    <zorder>enter_button</zorder>
    <zorder>title_input</zorder>
    <zorder>title_label_2</zorder>
    <zorder>id_input</zorder>
    <zorder>name_label</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
